### ISSUE TITLE
Item frames are not rendered at all

### ISSUE THREAD

**Issue by wolfen351** [overviewer/Minecraft-Overviewer#1830]
Please have a look at the screenshot, on all the Red Concrete blocks there are item frames with a sample of what is in the chest:

![image](https://user-images.githubusercontent.com/5477547/89701299-8c8cb500-d989-11ea-9242-d3e7398ed4fb.png)

In-Game:

![image](https://user-images.githubusercontent.com/5477547/89701318-b3e38200-d989-11ea-9e1c-665936acd5c4.png)

**Comment by wolfen351** [comment]
As a consequence, my map wall also doesn't render - the item frame and the map inside it don't render in the map:

![image](https://user-images.githubusercontent.com/5477547/89701336-02911c00-d98a-11ea-97c5-48b2f3e97291.png)

**Comment by Phoenix616** [comment]
I feel like this would be pretty hard to implement seeing as item frames are entities and there is currently no entity support besides player logout positions in Overviewer.

**Comment by LeadMagnet** [comment]
I agree with @Phoenix616, this would be an major undertaking to render the content of the item frame.

I'm not sure, but I think item frames are not render at all, regardless of what they contain (even empty). Which if you think about it makes perfect sense. Overviewer is supposed to give an "overview" of what is actually there, not a 100% recreation.
